### PR TITLE
CI - add kubetest

### PR DESCRIPTION
**Pull request by guidoiaquinti**
## Description
This PR adds a new type of test suite using `kubetest` (you can find more info about it in the `README.md` that I've added to this PR).

This is a subset of https://github.com/PostHog/charts-clickhouse/pull/224 that I'm splitting into multiple parts.

## Type of change
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## How has this been tested?
CI should be passing

## Checklist
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: ci/kubetest/test_placeholder.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import pytest
```
Comment (guidoiaquinti) [inline ci/kubetest/test_placeholder.py:1]: This will be removed when real test will be added

File: ci/kubetest/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Kubetest
+This directory contains the integration tests for our Helm chart. We use [`kubetest`](https://kubetest.readthedocs.io/en/latest/) that is a `pytest` plugin that makes it easier to write integration tests on Kubernetes.
+x
```
Comment (hazzadous) [inline ci/kubetest/README.md:3]:
```suggestion
```

**Review by hazzadous — APPROVED** [review]
Looks good

Non-blocking: I might be missing some context, but what does kubetest give us that we'd not handling with e.g. helm test, or just using `kubectl`?

**Comment by guidoiaquinti** [comment]
> Non-blocking: I might be missing some context, but what does kubetest give us that we're not able to handle with e.g. helm test, or just using `kubectl`?

Good question! I need to add more documentation about the difference but the tldr is the following:

- `helm lint`: lint test suite to verify if the template can be rendered
- `helm unittest`: unit test suite to verify if `rendered template == expected output`
- `kubetest`: integration test suite to verify if _applying the expected output_ against a k8s target cluster gives us also _the expected infrastructure_ (e.g. are the disk really encrypted? Is this environment variable set as expected?)

To see some `kubetest` test scenarios see https://github.com/PostHog/charts-clickhouse/pull/224/files

**Pull request merged**